test/integration/specs/load-twitter-component.js

```diff
@@ -137,7 +133,7 @@ describe('Twitter Component - Integration', () => {
       browser.waitForText(retweetsSelector);
       browser.getText(retweetsSelector).should.be.equal(expectedRetweets.toString());
   });
-
+  /*
```

Comment:
@rodrigopavezi  Had to comment out this test for now due to LMS issues as discussed on Hipchat.